index.user.js

```diff
@@ -12,6 +12,95 @@
 // @require      https://code.jquery.com/jquery-3.3.1.slim.min.js
 // ==/UserScript==
 
+var moomoohaxored;
+if (moomoohaxored) {
+    console.log("moomoo haxored")
+}
+
+(function() {
+
+var infinity = infinity;
+var neginfinity = -infinity;
```

Comment:
Ironically, this variable is useless as `neginfinity` has more bytes than `-Infinity`. The other variable is completely useless.